components/Chat/ChatInput.vue

```diff
@@ -351,45 +351,24 @@
           .match( /:[\w]*$/g );
 
         if ( emoteMatch ) {
-          this.autocompleteData = this.emoteList;
+          this.autocompleteData = Array.from( this.emoteMap.values() );
```

Comment:
I'm wondering if creating a new array each time is efficient compared to just storing an extra copy. Seems like it has the potential to create a lot of garbage if this creates a new ~500 item array each time this function is called.